torch/quantization/quantize_fx.py

```diff
@@ -289,6 +289,15 @@ def prepare_fx(
         ("foo.*bar.*conv[0-9]+", qconfig?)
         ...,
       ],
+
+      # optional, used for matching function invocations in a submodule by order
+      # TODO(future PR): potentially support multiple indices ('0,1') and/or
+      #   ranges ('0:3').
+      "module_name_function_order": [
+        # fully_qualified_name, function_type, index, qconfig
+        ("foo.bar", torch.nn.functional.linear, 0, qconfig?),
+      ],
+
       # priority (in increasing order): global, object_type, module_name_regex, module_name
```

Comment:
nit: please also remember to include the precedence in this list as well, we probably want to have two precedence list: one for module and one for function to make it more clear. (can be done in a separate PR)

Reply:
thanks, will fix this